lib/rubocop/cop/lint/interpolated_string.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for interpolation in a single quoted string.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #  foo = 'something with #{interpolation} inside'
```

Comment:
Code examples need an additional space of indentation to be in line with `# bad` and `# good`. 😊